app/views/katello/api/v2/docker_tags/_base.json.rabl

```diff
@@ -18,8 +18,26 @@ child :docker_manifest => :manifest do
   attributes :schema_version, :digest, :manifest_type
 end
 
-child :repositories => :repositories do
```

Comment:
Can we retain the `:repositories` list even if no organization is specified? Asking since this will be a breaking API change.

Reply:
Good point..Updated..